catroid/src/test/java/org/catrobat/catroid/test/formulaeditor/FormulaEditorTestUtil.java

```diff
@@ -135,6 +138,19 @@ public static void testDoubleParameterFunction(Functions function, InternTokenTy
 		assertEquals(expected, parseTree.interpretRecursive(testSprite));
 	}
 
+	public static void testDoubleParameterFunction(Functions function, InternTokenType firstInternTokenType,
+			String firstParameter, InternTokenType secondInternTokenType, String secondParameter, Double median,
+			Double error, Sprite testSprite) {
+
+		List<InternToken> internTokenList = buildDoubleParameterFunction(function, firstInternTokenType,
+				firstParameter, secondInternTokenType, secondParameter);
+		FormulaElement parseTree = new InternFormulaParser(internTokenList).parseFormula();
+
+		assertNotNull(parseTree);
+		Double result = (double) parseTree.interpretRecursive(testSprite);
+		assertThat(result, closeTo(median, error));
```

Comment:
this is fine, just so you know, assertEquals also supports delta for float and doubles.
so
assertEquals(doubleExpected, doubleActual, someSmallEpsilon)

Reply:
yes, right. forgot about that.